x-pack/plugins/snapshot_restore/public/app/types/app.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { FormattedMessage } from '@kbn/i18n/react';
+import chrome from 'ui/chrome';
+import { I18nContext } from 'ui/i18n';
+import { management, MANAGEMENT_BREADCRUMB } from 'ui/management';
+import { fatalError } from 'ui/notify';
```

Comment:
Have we checked with the platform team that it makes sense to use webpack aliases to import these things in a non-shim file? My understanding was that we wanted to isolate all external imports within the shim, but this seems contrary to that guidance.

Reply:
good call CJ, we shouldn't be importing these outside of `shim.ts`. maybe we can move these back into shim but re-export them here